src/pages/DriverDash.js

```diff
@@ -0,0 +1,124 @@
+                  <img class="card-img-top" src={require('../media/Larry.png')} alt="Card image"></img>
+              </div>
+              <div className = "text_box" style={{display:'inline-block','vertical-align':'left'}}>
+                      <p className="card-text" align="top" style={{'margin-top':'10px'}}><strong>Age: </strong>{json['user1'].age}</p>
+                      <p className="card-text" align="top"><strong>License Number: </strong>{json['user1'].license}</p>
+                      <p className="card-text" align="top"><strong>Plate Number: </strong>{json['user1'].plate}</p>
+                      <p className="card-text" align="top"><strong>Insurance Provider: </strong>{json['user1'].insurance}</p>
+              </div>
+            </div>
+          </div>
+		  
+        </div>
+            </div>
+            <div className="col">
+            <div class="row">
+            <div class="col text-center mb-3">
+              <a href='/create' className="btn btn-dark">
+                Create New Listing
+              </a>
+            </div>
+          </div>
+            <div className="card text-left w-75 m-3 mx-auto">
+            <h3 className="card-header"><strong>Trip: </strong>{json['trip1'].tripID}</h3>
+            <div className="card-body">
+              <p className="card-text"><strong>Date: </strong>{json['trip1'].date}</p>
+              <p className="card-text"><strong>Time: </strong>{json['trip1'].time}</p>
+              <p className="card-text"><strong>Departure: </strong>{json['trip1'].departure}</p>
+              <p className="card-text"><strong>Destination: </strong>{json['trip1'].destination}</p>
+              <p className="card-text"><strong>Spots Open: </strong>{json['trip1'].spotsOpen}</p>
+              <div class="row">
+                <div class="col text-center">
+                  <a href="/trip" className="btn btn-dark">
+                    View Trip Details
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+            <div className="card text-left w-75 m-3 mx-auto">
+            <h3 className="card-header"><strong>Trip: </strong>{json['trip2'].tripID}</h3>
+            <div className="card-body">
+              <p className="card-text"><strong>Date: </strong>{json['trip2'].date}</p>
+              <p className="card-text"><strong>Time: </strong>{json['trip2'].time}</p>
+              <p className="card-text"><strong>Departure: </strong>{json['trip2'].departure}</p>
+              <p className="card-text"><strong>Destination: </strong>{json['trip2'].destination}</p>
+              <p className="card-text"><strong>Spots Open: </strong>{json['trip2'].spotsOpen}</p>
+              <div class="row">
+                <div class="col text-center">
+                  <a href="/trip" className="btn btn-dark">
+                    View Trip Details
+                  </a>
+                </div>
```

Comment:
Should be "View Ride Requests" and href should be "/ride-requests-1" to render the ViewRequests component, not the Trip component.